src/integrations/govdata.ts

```
@@ -0,0 +1,66 @@
+}
+
+const sanitizeUrl = (raw?: string): string | null => {
+  try {
+    const u = new URL(raw ?? '');
+    return u.protocol === 'https:' ? u.toString() : null;
+  } catch {
+    return null;
+  }
+};
+
+    url: sanitizeUrl(pkg.url ?? pkg.resources?.[0]?.url)
```

Biome (2.1.2)

[error] Do not use the url variable name as a label

The variable is declared here

Creating a label with the same name as an in-scope variable leads to confusion.

(lint/suspicious/noLabelVar)